From f038511a5cfde15f0d9b167da3a3a0502174a03f Mon Sep 17 00:00:00 2001
From: kszaq <kszaquitto@gmail.com>
Date: Sun, 26 Feb 2017 16:59:58 +0100
Subject: [PATCH] aml: do not set blackout policy

Some decoders in amports look at blackout_policy when determining cma release policy. Do not play with policy to not confuse them.
---
 xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
index c38c223..13e843b 100644
--- a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
+++ b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
@@ -1741,11 +1741,6 @@ void CAMLCodec::Reset()
   if (!m_opened)
     return;
 
-  // set the system blackout_policy to leave the last frame showing
-  int blackout_policy;
-  SysfsUtils::GetInt("/sys/class/video/blackout_policy", blackout_policy);
-  SysfsUtils::SetInt("/sys/class/video/blackout_policy", 0);
-
   // restore the speed (some amcodec versions require this)
   if (m_speed != DVD_PLAYSPEED_NORMAL)
   {
@@ -1763,9 +1758,6 @@ void CAMLCodec::Reset()
   am_private->am_pkt.codec = &am_private->vcodec;
   pre_header_feeding(am_private, &am_private->am_pkt);
 
-  // restore the saved system blackout_policy value
-  SysfsUtils::SetInt("/sys/class/video/blackout_policy", blackout_policy);
-
   // reset some interal vars
   m_1st_pts = 0;
   m_cur_pts = 0;
-- 
1.8.3.1

